### PR TITLE
Add a workflow to break a pacman upload lease

### PR DESCRIPTION
**Pull request by rimrul**
This fixes https://github.com/git-for-windows/git-for-windows-automation/issues/40

### CODE REVIEW

**Comment by rimrul** [comment]
This could probably almost run on `ubuntu-latest` runner, but we use `cygpath` in these scripts.

File: .github/workflows/break-pacman-upload-lease.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  workflow_dispatch:
+
+jobs:
+  break-lease:
+    runs-on: windows-latest
+    steps:
+      - uses: git-for-windows/setup-git-for-windows-sdk@v1
+        with:
+          flavor: minimal
+          architecture: x86_64
```
Comment (dscho) [inline .github/workflows/break-pacman-upload-lease.yml:14]: Do we even need the minimal SDK? As far as I can tell, [`pacman-helper.sh break_lock`](https://github.com/git-for-windows/build-extra/blob/HEAD/pacman-helper.sh) simply hands off to [`wingit-snapshost-helper.sh`](https://github.com/git-for-windows/build-extra/blob/HEAD/wingit-snapshot-helper.sh), which calls `curl`. GitHub workflows' `bash` steps use Git for Windows' Bash (!!!), which is distributed with a working `curl`, so it _should_ work out of the box.

**Review by dscho — APPROVED** [review]
Nice, thank you!

I just have one question regarding the need to set up a minimal Git for Windows SDK. It's a minor concern, though, I'd be fine with merging as-is.

File: .github/workflows/break-pacman-upload-lease.yml (hunk 1 above)
Comment (rimrul) [reply .github/workflows/break-pacman-upload-lease.yml:14]: We also use `date`, `xxd`, `tr`, `sed`, `base64` and `cygpath`, but that should work.

**Comment by dscho** [comment]
@rimrul I opened https://github.com/git-for-windows/build-extra/pull/566 (ran it in Linux to verify that it works; It said "409 There is currently no lease on the blob." of course 😁); How do you feel about merging that first and then switching this here GitHub workflow to `ubuntu-latest`?

**Review by dscho — APPROVED** [review]
Yeah!

**Pull request merged**